Mathlib/Algebra/Invertible.lean

```diff
@@ -76,7 +76,7 @@ class Invertible [Mul α] [One α] (a : α) : Type u where
 #align invertible Invertible
 
 /-- The inverse of an `Invertible` element -/
-notation:1034
+prefix:max
```

Comment:
Note that this change makes this notation more closely match the `Inv.inv` notation: https://github.com/leanprover-community/mathlib4/blob/cac405f4bb86f998a4d71cb6836b2a05afb4bcea/Mathlib/Algebra/Group/Defs.lean#L111

and it makes the comment on the next line more true.